src/Bridge/Symfony/Bundle/Resources/config/doctrine_orm_mercure_publisher.xml

```diff
@@ -8,7 +8,7 @@
 
         <!-- Event listener -->
 
-        <service id="api_platform.doctrine.listener.mercure.publish" class="ApiPlatform\Core\Bridge\Doctrine\EventListener\PublishMercureUpdatesListener">
+        <service id="api_platform.doctrine.orm.listener.mercure.publish" class="ApiPlatform\Core\Bridge\Doctrine\EventListener\PublishMercureUpdatesListener">
```

Comment:
It's a BC break isn't it? We should at least add a (deprecated) alias to the old service name.

Reply:
Even if it's experimental?

Reply:
Please, yes. The class is marked as experimental but not the definition, and I know that Arte rely on this for their doctrine-less implementation.

Reply:
And it doesn't hurt. It's just one line to add, that will be dropped in 3.0.